common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -416,6 +417,303 @@ static std::shared_ptr<char> generate_client_assoc_event(const std::string &even
     return msg_buff;
 }
 
+// Checks if file exists
+static bool file_exists(const std::string &fname)
+{
+    struct stat st;
+    return (stat(fname.c_str(), &st) == 0);
+}
+
+// Finds value by key in a string vector representing hostapd config (head or VAP)
+// and returns value found in the supplied string object.
+static bool hostapd_config_get_value(const std::vector<std::string> &hostapd_config,
+                                     const std::string &key, std::string &value)
+{
+    std::string key_eq(key + "=");
+    auto it_str =
+        std::find_if(hostapd_config.begin(), hostapd_config.end(), [&](std::string str) -> bool {
```

Comment:
There is no need to pass everything as a reference to the lambda function. Please move only `&key_eq`.